src/runtime_src/core/common/sensor.cpp

```diff
@@ -64,14 +66,16 @@ populate_sensor(const xrt_core::device * device, const std::string& loc_id, cons
     pt.put("current.error_msg", ex.what());
   }
   pt.put("current.amps", xrt_core::utils::format_base10_shiftdown3(current));
-  pt.put("current.is_present", current != 0 ? "true" : "false");
-  
+  pt.put("current.is_present", current != 0 ? true : false);
```

Comment:
As per my knowledge, this true and false are kept as string format for json output. Better not to change these to boolean. (all other places too.)